__std__/nix/toolchain/packages/pre-commit-check.nix

```diff
@@ -0,0 +1,40 @@
+{ inputs, cell }:
+
+cell.packages.todo-derivation
+
+# TODO(std) we need stylish-haskell for this
```

Comment:
could just comment out the lines between 10 and 15 for the tools we don't have yet

Reply:
👍